spec/models/chat_message_spec.rb

```diff
@@ -109,7 +109,7 @@
     it 'supports emoji plugin' do
       cooked = ChatMessage.cook(":grin:")
 
-      expect(cooked).to eq("<p><img src=\"/images/emoji/twitter/grin.png?v=12\" title=\":grin:\" class=\"emoji only-emoji\" alt=\":grin:\"></p>")
+      expect(cooked).to eq("<p><img src=\"/images/emoji/twitter/grin.png?v=12\" title=\":grin:\" class=\"emoji only-emoji\" alt=\":grin:\" loading=\"lazy\" width=\"20\" height=\"20\"></p>")
```

Comment:
Oh this is interesting, what caused this?

Reply:
O this is from a recent commit where we cook emojis differently.